presto-docs/src/main/sphinx/sql/select.rst

```diff
@@ -1061,7 +1061,14 @@ standard rules for nulls. The subquery must produce exactly one column::
 
     SELECT name
     FROM nation
-    WHERE regionkey IN (SELECT regionkey FROM region)
+    WHERE regionkey IN (SELECT regionkey FROM region
+    WHERE name = 'AMERICA' OR name = 'AFRICA');
```

Comment:
Can you indent the subquery so that the two `WHERE` clauses are not aligned? It looks funny and is hard to read that way. How about
```sql
SELECT name
FROM nation
WHERE regionkey IN (
    SELECT regionkey
    FROM region
    WHERE name = 'AMERICA' OR name = 'AFRICA'
);

Reply:
Or we could shorten it so that it fits on one line
```sql
WHERE regionkey IN (SELECT regionkey FROM region WHERE name = 'AFRICA')
```